test-suite/shared/src/test/require-jdk11/testsuite/javalib/lang/StringTestJDK11.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.lang
+
+import java.nio.charset.Charset
+
+import scala.language.implicitConversions
+
+import org.junit.Test
+import org.junit.Assert._
+
+import org.scalajs.testsuite.utils.AssertThrows._
+import org.scalajs.testsuite.utils.Platform._
+
+class StringTestJDK11 {
```

Comment:
To follow the (dominating) convention, this should be called `StringTestOnJDK11`:
```suggestion
class StringTestOnJDK11 {
```
(and the file name should be adapted)